resource_lists/stardew valley/resources.yaml

```diff
@@ -0,0 +1,7165 @@
+      - output: 1
+        recipe_type: As
+        requirements:
+          Octopus Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Oyster Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Perch Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Periwinkle Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Pike Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Pufferfish Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Rainbow Trout Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Red Mullet Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Red Snapper Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Salmon Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Sandfish Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Sardine Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Scorpion Carp Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Sea Cucumber Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Sea Urchin Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Shad Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Shrimp Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Slimejack Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Smallmouth Bass Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Snail Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Spook Fish Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Stingray Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Stonefish Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Sunfish Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Super Cucumber Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Tiger Trout Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Tilapia Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Tuna Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Void Salmon Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Walleye Roe: -1
+      - output: 1
+        recipe_type: As
+        requirements:
+          Woodskip Roe: -1
+
+  Any Wild Seeds:
```

Comment:
Eventually this will be a resource group, but that feature has lacking functionality. Should this be a resource group now, or wait until later when that feature works better?

Reply:
My thought is that since some wild seeds are easier to craft than others it is likely that a particular type is chosen if wild seeds are being used in a planned out bulk crafting, so for now having them like this allows for that.